src/collectHeatmapClicksVirtualTag.js

```diff
@@ -0,0 +1,101 @@
+((document, window) => {
+  const PPASHeatmapClickEvent = 'PPASHeatmapClickProxyEvent';
+
+  const getElementPath = (event, { CSSClassesSeparators }) => {
+    const replaceClassnameParts = (classes) => {
+      let newClasses = classes;
+      CSSClassesSeparators.forEach(rule => {
+        newClasses = newClasses.replace(new RegExp(rule, 'g'), ' ').trim();
+      });
+
+      return newClasses;
+    };
+
+    const getRawStringPath = (steps) => {
+      let tag, stringSteps = [];
+      for (let i = 0, element = steps[i]; element; element = element.parentNode, i++) {
+        if (element === document || element === window) break;
+
+        tag = element.tagName.toLowerCase();
+
+        if (tag === 'html') break;
+
+        // cut path when id exists
+        if (element.id) {
+          tag = `#${element.id}`;
+          stringSteps.push(tag);
+          break;
+        }
+
+        // if parent has multiple same tags so we should defined which child it is 
+        if (Array.prototype.slice.call(element.parentNode.children).filter(child => child.tagName.toLowerCase() === tag).length > 1) {
+          const allSiblings = Array.prototype.slice.call(element.parentNode.children);
+          const index = allSiblings.indexOf(element) + 1;
+          if (index > 0) {
+              tag += `:nth-child(${index})`;
+          }
+        }
+
+        // exclude SVGAnimatedString className objects
+        if (typeof element.className === 'string' && element.className) {
+          let classes = element.className.trim();
+          if (CSSClassesSeparators.length) {
+            classes = replaceClassnameParts(classes);
+          }
+          classes = classes.replace(/\s+/g, '.');
+          tag += `.${classes}`;
+        }
+
+        stringSteps.push(tag);
+      }
+
+      return stringSteps.reverse().join('>');
+    };
+    let targets = [];
+    let finalPath = '';
+
+    // handling missing method in IE11 / Edge
```

Comment:
```suggestion
    // handling missing method in IE11
```
Edge supports `event.composedPath`